src/controllers/expensesController.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const expensesServices = require('../services/expensesServices.js');
+const userServices = require('../services/userServices.js');
+
+const getAll = (req, res) => {
+  res.status(200).send(expensesServices.getAll(req.query));
+};
+
+const add = (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  if (!userId
+    || !spentAt
+    || !title
+    || !amount
+    || !category
+    || !note
+    || !userServices.getUserById(userId)) {
```

Comment:
Long expression add to variables. Improve all cases

Reply:
I thought for a long time. No ideas how to do this